src/components/CheckExtenstionWalletProvider/index.js

```diff
@@ -41,6 +40,16 @@ export function CheckExtensionWalletProvider({ children }) {
     || wallets === null
     || isUnsupportedBrowser === null;
 
+  useEffect(() => {
+    if (!walletAddress && (userBlockchainAdressStorage || (wallets?.length > 0 && wallets[0]?.address))) {
+      dispatch(
+        blockchainActions.setUserWallet.success(
+          userBlockchainAdressStorage || wallets[0].address,
```

Comment:
hmmm will have to see how this works with wallets that require you to connect again to the site